examples/onnxrt/nlp/huggingface_model/text_generation/llama/quantization/ptq_static/main.py

```diff
@@ -0,0 +1,272 @@
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint:disable=redefined-outer-name,logging-format-interpolation
+import os
+import onnx
+import torch
+import logging
+import argparse
+import numpy as np
+from datasets import load_dataset
+import onnxruntime as ort
+from torch.nn.functional import pad
+from torch.utils.data import DataLoader
+from intel_extension_for_transformers.evaluation.lm_eval import evaluate
+from optimum.onnxruntime import ORTModelForCausalLM
+from transformers import LlamaConfig, LlamaTokenizer
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(format = '%(asctime)s - %(levelname)s - %(name)s -   %(message)s',
+                    datefmt = '%m/%d/%Y %H:%M:%S',
+                    level = logging.WARN)
+
+parser = argparse.ArgumentParser(
+formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument(
+    '--model_path',
+    type=str,
+    help="Folder path of pre-trained onnx model"
+)
+parser.add_argument(
+    '--benchmark',
+    action='store_true', \
+    default=False
+)
+parser.add_argument(
+    '--tune',
+    action='store_true', \
+    default=False,
+    help="whether quantize the model"
+)
+parser.add_argument(
+    '--output_model',
+    type=str,
+    default=None,
+    help="output model path"
+)
+parser.add_argument(
+    '--mode',
+    type=str,
+    help="benchmark mode of performance or accuracy"
+)
+parser.add_argument(
+    '--batch_size',
+    default=1,
+    type=int,
+)
+parser.add_argument(
+    '--tokenizer',
+    type=str,
+    help="pretrained model name or path of tokenizer files",
+    default="decapoda-research/llama-7b-hf"
+)
+parser.add_argument(
+    '--workspace',
+    type=str,
+    help="workspace to save intermediate files",
+    default="nc_workspace"
+)
+parser.add_argument(
+    '--quant_format',
+    type=str,
+    default='QOperator', 
+    choices=['QOperator', 'QDQ'],
+    help="quantization format"
+)
+parser.add_argument(
+    '--pad_max',
+    default=196,
+    type=int,
+)
+parser.add_argument(
+    "--tasks",
+    nargs='+',
+    default=["winogrande", "copa", "piqa", "rte", "hellaswag", "openbookqa", \
+             "lambada_openai", "lambada_standard", "wikitext"],
+    type=str,
+    help="tasks list for accuracy validation"
+)
+parser.add_argument(
+    "--dataset",
+    nargs="?",
+    default="NeelNanda/pile-10k",
+    const="NeelNanda/pile-10k"
+)
+parser.add_argument(
+    "--smooth_quant_alpha",
+    type=float,
+    default=0.6
+)
+parser.add_argument(
+    "--intra_op_num_threads",
+    type=int,
+    default=4
+)
+args = parser.parse_args()
+
+# load model
+tokenizer = LlamaTokenizer.from_pretrained(args.tokenizer)
```

Comment:
Wouldn't it fail here because run_quant.sh set `tokenizer` to `${input_model}`? The tokenizer file is not included in `${input_model}`